src/Type/Definition/FieldArgument.php

```diff
@@ -32,6 +32,13 @@ class FieldArgument
     /** @var InputType&Type */
     private $type;
 
+    /**
+     * Helps to differentiate when `defaultValue` is `null` and when it was not even set initially
+     *
+     * @var bool
+     */
+    private $defaultValueExists = false;
```

Comment:
Why is this making a return?

Reply:
This change is already reverted.

The implementation for those kinds of checks was unified in https://github.com/webonyx/graphql-php/pull/585.